docs/Installation.md

```diff
@@ -3,7 +3,10 @@
 The ML-Agents Toolkit contains several components:
 
 - Unity package ([`com.unity.ml-agents`](../com.unity.ml-agents/)) contains the
-  Unity C# SDK that will be integrated into your Unity scene.
+  Unity C# SDK that will be integrated into your Unity scene.  This package contains
+  the [3DBall example](../Project/Assets/ML-Agents/Examples/3DBall/Scenes/3DBall.unity)
```

Comment:
Can we just say something generic like "The package contains samples to show you how to get started with ML-Agents"? Otherwise we need to maintain the list here, and I think it distracts from the instructions.

If you want to call out the forum thread, can you link to it here instead? https://github.com/Unity-Technologies/ml-agents#community-and-feedback

Reply:
sure